src/Assets/Scripts/Managers/OnObjectClickManager.cs

```diff
@@ -238,5 +241,18 @@ public void DestroySelectedObject()
 				StartCoroutine(ApiManager.Instance.KillVisualizedObject(_selectedObject, true));
 			InfoPanel.SetActive(false);
 		}
+
+		public void OpenSelectedObjectUrl()
+		{
+			if (!(_selectedObject is IVisualizedBuilding)) return;
+
+			NeighbourhoodModel neighbourhoodModel = CityManager.Instance.GameModel.Neighbourhoods
+				.Select(x => x).SingleOrDefault(x =>
```

Comment:
Good that you use SingleOrDefault instead of FirstOrDefault, but make sure this does find an object. Otherwise it crashes!